scss/components/_navbar.scss

```diff
@@ -1,53 +1,9 @@
-$primary-color: #754ffe;
-$secondary-color: #6343d8;
-$white-default: #ffffff;
-$black-default: #000000;
-$box-shadow-color: #cccccc;
-$item-grey: #64748b;
-$bg-hover: #f1f5f9;
-
-* {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-}
-
-html {
-    font-family: 'Inter', sans-serif;
-    font-size: 10px;
-}
-
-body {
-    background: $white-default;
-}
- 
-@mixin flexbox($flex-direction: row, $justify: center, $align: center) {
-    display: flex;
-    flex-direction: $flex-direction;
-    justify-content: $justify;
-    align-items: $align;
-}
-
-@mixin btn-primary($border-radius: 0.3rem) {
-    padding: 1.5rem;
-    width: 13rem;
-    background-color: $primary-color;
-    color: $white-default;
-    border: none;
-    border-radius: $border-radius;
-    font-weight: 600;
-    transition: background-color 0.3s ease;
-    cursor: pointer;
-
-    &:hover {
-        background-color: $secondary-color;
-    }
-}
+@use '../abstracts' as abs;
 
 .navbar {
-    @include flexbox(row, space-evenly, center);
+    @include abs.flexbox(row, space-evenly, center);
```

Comment:
This might be beneficial in some more complex applications. But for this one, refer to what I wrote in _main.scss section.